### DISCUSSION TITLE
Show the code button on files

### DISCUSSION THREAD

**Discussion by reeceatkinson** [community/community#4827] [Codespaces] [answered]
The Code button currently displayed on the first screen should be shown on files and different directories.

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @reeceatkinson, thanks for the feedback. Can you please share a bit more context around the request to help me capture this in our backlog?

**Reply by reeceatkinson** [reply]
![image](https://user-images.githubusercontent.com/78908925/128205435-9d6f120d-f268-47cb-a974-ed33912f530a.jpeg)

The highlighted button above should be in all directories
e.g when opening a codespace and you just found the file you want but have to go back t9 the beginning

**Reply by tanmayeekamath** [reply]
Thanks, we will log this in our backlog. :)
